tools/bin/ariaNLCDmask.py

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env python3
```

Comment:
move this code as pythin code and keep a bin equivalent that only has an argparse. e.g. ariaTSprep.py, ariaExtract.py